pyproject.toml

```diff
@@ -53,22 +53,24 @@ keywords = [
   ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<4.0"  # Cannot remove upper-bound as isort needs >=3.6.1,<4.0
+python = ">=3.8.1,<4.0"  # Cannot remove upper-bound as isort needs >=3.6.1,<4.0
 Django = ">=2.2"
-django-crispy-forms = ">=1.13.0"
+django-crispy-forms = ">=1.13.0,<2.0"
 
 [tool.poetry.dev-dependencies]
 pre-commit = "2.15.0"
 beautifulsoup4 = ">=4.8,<4.10.0"
 coverage = "6.1.1"
 pymdown-extensions = "9.0"
 poetry = "1.1.13"
+syrupy = "^4.0.1"
```

Comment:
Suggest this is pinned to prevent issues with different devs running different versions